src/Http/Http.Abstractions/src/HttpMethods.cs

```diff
@@ -18,18 +18,45 @@ public static class HttpMethods
         // and allow us to optimize comparisons when these constants are used.
 
         // Please do NOT change these to 'const'
+        /// <summary>
+        /// Constant for HTTP "CONNECT" method.
```

Comment:
What do we do elsewhere in our source for readonly string values like these?

I'm not sure that constant is the right word. The property is readonly and not constant.

Reply:
TBH, most of the read-only strings are undocumented. Those that are seem to just describe the property directly (e.g. HTTP "CONNECT" method). I can update this accordingly.

Reply:
Fixed here and elsewhere.